### ISSUE TITLE
Validation Error

### ISSUE THREAD

**Issue by jenjenniee** [jenjenniee/NLP_Project#2]
<img width="446" alt="file-QxHHbEqYNhDdBRbQF3hcowjy" src="https://github.com/user-attachments/assets/3cf81ed7-f64b-416d-bd40-25768a4a6480">

### 이슈 주제
Training loss는 줄어드나, Validation loss는 일정한 형태를 보임

### Why?
Validation 데이터가 잘못 전달되었거나, 학습 데이터와 거의 동일하여 모델이 Validation에서 낮은 loss가 발생할 수 있음

**Comment by jenjenniee** [comment]
### 해결 방법
1. Validation 데이터셋의 분포 확인 
`print(f"Validation Dataset Size: {len(valid_dataset)}")
print("Sample Validation Data:", valid_dataset[0])`
![IMG_7255](https://github.com/user-attachments/assets/1d1d02a1-13ac-48a8-b171-4d91ee715751)
input_ids 텐서의 상당 부분이 0으로 패딩된 상태임을 알 수 있었다.
따라서, 패딩된 데이터셋의 전처리 및 제거 후 Attention Mask 확인 및 재설정하고 다시 학습을 실행해 보려고 함.
